docs: openssh updates (#28438)
gravitational/teleport: docs/pages/server-access/guides/openssh.mdx

Reviewer: No, good find!

Author: Yes that looks incorrect, please look update that is inline with other examples.

Teleport only allows access to resources in your infrastructure via Teleport
processes that that have joined the cluster.

To register the OpenSSH node requires a valid auth token to connect to the cluster.
Generate one by running the following command against your Teleport Auth Service:

```code
$ tctl tokens add --type=node --format=text
(=presets.tokens.first=)
```

Copy the `teleport` binary to your `sshd` host. Configure `sshd` and create a node

> resource on your cluster with the following command, assigning <Var name="teleport.example.com:443" />
> to the address of your Teleport Proxy Service, <Var name="server1.example.com:22" />
> to the address of the node that will join the cluster, and <Var name="token" /> to the token value:

Reviewer 2: ```suggestion
resource on your cluster with the following command, assigning
<Var name="teleport.example.com:443" /> to the address and port of your Teleport
Proxy Service, <Var name="server1.example.com:22" /> to the address and port of
the node that will join the cluster, and <Var name="token" /> to the token value:
```

Adjusted to mention port as well as address, and for line breaks ~80 chars.

Teleport only allows access to resources in your infrastructure via Teleport
processes that that have joined the cluster.

> To register the OpenSSH node requires a valid auth token to connect to the cluster.

Reviewer 3: ```suggestion
To register the OpenSSH node, you must have a valid auth token to connect to the cluster.
```

> Generate one by running the following command against your Teleport Auth Service:

Reviewer 3: ```suggestion
You can generate the token by running the following command against your Teleport Auth Service:
```

```code
$ tctl tokens add --type=node --format=text
(=presets.tokens.first=)
```

Copy the `teleport` binary to your `sshd` host. Configure `sshd` and create a node

> resource on your cluster with the following command, assigning

Reviewer 3: I might suggest splitting this into a statement and list. For example:
Copy the `teleport` binary to your `sshd` host. Configure `sshd` and create a node 
resource on your cluster with the following command:

```
command
```

Change the command-line options to assign the following values:
- Set <Var name="teleport.example.com:443" /> to the address and port of your Teleport
Proxy Service.
- Set <Var name="server1.example.com:22" /> to the address and port of
the node that will join the cluster.
- Set <Var name="token" /> to the token value.

<Var name="teleport.example.com:443" /> to the address and port of your Teleport
Proxy Service, <Var name="server1.example.com:22" /> to the address and port of
the node that will join the cluster, and <Var name="token" /> to the token value:

```code
$ sudo teleport join openssh \
  --address <Var name="server1.example.com:22" /> \
  --proxy-server <Var name="teleport.example.com:443" /> \
  --join-method token \
  --token <Var name="token" /> \
  --labels env=dev
```

Note that both the Teleport Proxy Service and node addresses must contain a host and a port.
Check that your new node is listed with `tsh ls` or in the Web UI.  You can edit the

> hostname and labels with `tctl edit nodes/<hostname>`.  If the hostname isn't unique get the UUID

Reviewer 3: ```suggestion
hostname and labels with `tctl edit nodes/<hostname>`.  If the hostname isn't unique, get the UUID
```

and <Var name="node" /> to the name of the node that will join the cluster:

```code

> $ sudo teleport join openssh --address <Var name="node" /> --proxy-server <Var name="proxy" /> \

Reviewer: ```suggestion
$ sudo teleport join openssh \
  --address <Var name="node" /> \
  --proxy-server <Var name="proxy" /> \
```

I would break lines for each flag, rather than having 2 flags on the first line and one flag on all following lines

  --join-method token --token <TOKEN> \
  --labels env=dev
```

> Note that both the Teleport Proxy Service and node addresses must contain a host and a port.

Author: Yes, I've updated to that type of example. Also added token variable.
